src/app/shared/search/search-filters/search-filters.component.html

```diff
@@ -1,4 +1,4 @@
-<h3>{{"search.filters.head" | translate}}</h3>
+<h3>{{filterLabel+'.filters.head' | translate}}</h3>
```

Comment:
The current e2e tests are failing because of an accessibility issue here.  The error is: `'Ensures the order of headings is semantically correct',`

The reason for the error is that the accessibility scanner is expecting this to be an `<h2>` on the homepage.  But it **must** be an `<h3>` everywhere else.

I think we need to look into whether there's a way to make this an `<h2>` ONLY on the homepage, while it's kept as an `<h3>` everywhere else.

Reply:
<h3 *ngIf="inPlaceSearch">{{filterLabel+'.filters.head' | translate}}</h3>
<h2 *ngIf="!inPlaceSearch">{{filterLabel+'.filters.head' | translate}}</h2>